components/tooling/fetch-tests/test/src/main/java/mozilla/components/support/test/Expect.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.test
+
+import org.junit.Assert.fail
+import kotlin.reflect.KClass
+
+/**
+ * Expect [block] to throw an exception. Otherwise fail the test (junit).
+ */
+inline fun <reified T : Throwable> expectException(clazz: KClass<T>, block: () -> Unit) {
```

Comment:
🤔  I'm not quite sure why we're copying some classes from  `support-test` instead of bring them as a dependency?

Reply:
Ops, that was a copy error. I started implementing this in `support-test` and then moved it over to its own component later. :)

Reply:
👍